.github/copilot-instructions.md

```
@@ -0,0 +1,106 @@
+- **Upstream:** https://github.com/XOOPS/base-requires25
+- **Issues:** https://github.com/XOOPS/XoopsCore25/issues
+- **License:** GPL-2.0-or-later
+
+## Project Layout
+
+```
+composer.json                    # The sole deliverable — defines all dependencies
+LICENSE                          # GPL-2.0
+README.md                        # Usage instructions
+.github/
+  copilot-instructions.md        # This file
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)